src/app/forms/validators/custom-validators.ts

```diff
@@ -180,7 +180,10 @@ export class CustomValidators {
   }
 
   static email(): ValidatorFn {
-    return this.customPattern(['^[-\\w.\\+]+@[-\\w]+\\.[A-Za-z]{2,}$', 'is an invalid format. An email must include an @ symbol and a .']);
+    return this.customPattern([
+      '^[-\\w.\\+]+@[-\\w]+\\.[A-Za-z]{2,}$',
+      'is an invalid format. An email must include an @ symbol, include a full stop and end with two or more characters.'
```

Comment:
has `include` in it twice now... how about

```suggestion
      'is an invalid format. An email must include an @ symbol, a full stop and end with two or more characters'
```